Sources/BraintreeSEPADirectDebit/BTSEPADirectDebitClient.swift

```diff
@@ -69,4 +102,56 @@ import BraintreeCore
             completion(result, error)
         }
     }
+    
+    func startAuthenticationSessionWithoutContext(
```

Comment:
I was thinking we could add some documentation to these internal sepa functions since the flow won't be documented externally. Might help folks in the future when they are trying to figure out the order of this multi-step SEPA process.